app/assets/javascripts/discourse/app/components/discovery-topics-list.js

```diff
@@ -21,8 +21,19 @@ const DiscoveryTopicsListComponent = Component.extend(UrlRefresh, LoadMore, {
     }
   },
 
-  @observes("topicTrackingState.states")
-  _updateTopics() {
+  @on("didInsertElement")
+  _monitorTrackingState() {
+    this.stateChangeCallbackId = this.topicTrackingState.onStateChange(
+      this._updateTrackingTopics.bind(this)
```

Comment:
This is an aesthetic comment, but do you find `bind(this)` more readable, than, say:

```
this.topicTrackingState.onStateChange(() => this._updateTrackingTopics())
```

Reply:
agreed, fixed